src/tools/cephfs/cephfs-shell

```diff
@@ -773,6 +773,129 @@ class CephFSShell(Cmd):
 
         return 0
 
+    def complete_ln(self, text, line, begidx, endidx):
+        """
+        auto complete of file name.
+        """
+        return self.complete_filenames(text, line, begidx, endidx)
+
+    ln_parser = argparse.ArgumentParser(
+        description='Create a hard link or a symbolic '
+                    'link to an existing file or directory.')
+    ln_parser.add_argument('source_path', type=str, action=path_to_bytes,
```

Comment:
nit: opinion: would be easier if we have shorter name like `s_path`.

Reply:
s_path wouldn't be a bit short? it doesn't speak out what it actually is, maybe `source` can be used?

Reply:
On second thought what about just `source` and `dest`?

Reply:
`src` and `dest` look better. Will make the change

Reply:
Done

Reply:
Just `src` and `dest` since it's obvios these will be paths?

Reply:
And I forgot to post it yesterday, won't names `target` and `link` be more appropriate be  destination is actually not destination?

Reply:
`link` looks better but maybe i can use `source` instead of `target` ?